flagging_site/templates/base.html

```diff
@@ -11,14 +11,17 @@
     https://jinja.palletsprojects.com/en/2.11.x/templates/
 #}
 <html lang="en">
+
 <head>
     {% block head %}
     <title>
         {% block title %}{% endblock %} - CRWA Flagging Program
     </title>
     <link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}" />
+    <link rel="icon" type="image/ico" href="../static/favicon/favicon.ico">
```

Comment:
I would suggest revising this to use the Jinja convention of `{{ url_for('static', filename='favicon/favicon.ico') }}`.

 If you're unable to do that, I can still accept the PR and I'll make the change later.